etna/records/models.py

```diff
@@ -224,15 +224,28 @@ def catalogue_source(self) -> str:
 
     @cached_property
     def description(self) -> str:
-        if raw := self._get_raw_description():
+        """
+        Returns the records full description value with all HTML left intact.
+        """
+        return mark_safe(self._get_raw_description(use_highlights=False))
+
+    @cached_property
+    def listing_description(self) -> str:
+        """
+        Returns a version of the record's description that is safe to use
+        anywhere. When highlight data is provided by the API, <mark> tags
+        will be left in-tact, but and other HTML is stripped.
+        """
+        if raw := self._get_raw_description(use_highlights=False):
             return mark_safe(strip_html(raw, preserve_marks=True))
         return ""
 
-    def _get_raw_description(self) -> str:
-        try:
-            return "... ".join(self.highlights["@template.details.description"])
-        except KeyError:
-            pass
+    def _get_raw_description(self, use_highlights: bool = True) -> str:
+        if use_highlights:
```

Comment:
Happy for highlight use to be opt-in rather than opt-out... as it does have a significant impact on the return value. Fixed in [eef6f6f](https://github.com/nationalarchives/ds-wagtail/pull/730/commits/eef6f6f861fd14f458e1d5457c6fa123e450a4ac)